lib/active_merchant/billing/gateways/two_c_two_p.rb

```diff
@@ -45,9 +45,12 @@ def purchase(money, payment, options={})
         commit('sale', post)
       end
 
-      # payload is an encoded string, there is no way to scrub something there
       def supports_scrubbing?
-        false
+        true
+      end
+
+      def scrub(transcript)
+        force_utf8(transcript).gsub(/(paymentRequest=).+/, '\1[FILTERED]')
```

Comment:
This scrubs _everything_ right? it's a bit much because then we don't know _anything_ we've sent to the gateway ...